timeConversion_paircoded.js

```diff
@@ -0,0 +1,52 @@
+// Write a method that takes in a number of minutes, and returns a string that formats the number into hours:minutes.
+
+// input: number (in minutes)
+// output: string (in format hh:mm)
+
+// homework: if the minutes is smaller than 10 and a zero in front
+
+// cover edge cases:
+// negative number --> tell the user to only input positive number if they a negative one
+// string? --> tell user to only enter a number
+
+function timeConversion(minutes) {
+  // 1. find out how many hours are in the minutes
+  const hours = Math.floor(minutes / 60);
+
+  // 2. find the remaining minutes
+  const onlyMinutes = minutes % 60;
+  if (typeof (minutes) != Number) {
```

Comment:
double check how to write an if statement to check types (hint: log `typeof minutes` to the console and make sure you compare to that primitive type)